scylla/tests/integration/utils.rs

```diff
@@ -30,6 +31,31 @@ pub(crate) fn setup_tracing() {
         .try_init();
 }
 
+/// Finds the local IP address for a given destination IP address.
+///
+/// This function uses the `ip route get` command to get the routing information for the destination IP.
+pub(crate) fn find_local_ip_for_destination(dest: IpAddr) -> Option<IpAddr> {
+    let output = Command::new("ip")
+        .arg("route")
+        .arg("get")
+        .arg(dest.to_string())
+        .output()
+        .ok()?;
+
+    let output_str = String::from_utf8_lossy(&output.stdout);
```

Comment:
:question: Why not https://doc.rust-lang.org/std/str/fn.from_utf8.html?
```suggestion
    let output_str = std::str::from_utf8(&output.stdout).ok()?;
    
```